flake.nix

```diff
@@ -35,10 +35,49 @@
           (system:
             let
               pkgs = import nixpkgs { inherit system; };
+              extractRustAnalyzer = { stdenv, pkgs }: stdenv.mkDerivation {
+                pname = "extract_rust_analyzer";
+                version = "master";
+
+                dontUnpack = true;
+                dontBuild = true;
+
+                buildInputs = [ pkgs.python3 ];
+
+                installPhase = ''
+                  ls -la
+                  mkdir -p $out/bin
+                  cp ${./helpers/extract_rust_analyzer.py} $out/bin/extract_rust_analyzer.py
+                '';
+              };
+              extractRustAnalyzerPkg = pkgs.callPackage extractRustAnalyzer { };
             in
             {
-              packages.docs = pkgs.callPackage (import ./docs.nix) {
-                modules = nixvimModules;
+              packages = {
```

Comment:
not sure if flake.nix is the best place for this - is there any reason this can't just be in the rust-analyzer file? Only real reason I see to update flake.nix is to add inputs, I don't see how you'd need to use `extractRustAnalyzer` externally

Reply:
It's just quite easy to do right now, you run `nix run .#` and it generates the file, and it has access to the `pkgs` of the nixvim flake. I don't really see how to generate the .nix file else.

Reply:
Alternatively we could only expose the generating script in a nix devolp shell, but I feel it's more complex for not much.

Reply:
Ah I see what you mean, in that case I think this is fine

Reply:
Still, I'll rename the package to runUpdates so that we can extend this to update other stuff in the future